scripts/ci/travis/run-tests.sh

```diff
@@ -174,6 +174,7 @@ case "${CHAINER_TRAVIS_TEST}" in
             install)
                 run_prestep install_chainer_test_deps
                 run_prestep chainer_install_from_sdist
+                run_prestep chainer_example_tests
```

Comment:
shouldn't be `install_chainer_example_tests`?

Reply:
Oops... Force-pushed in f586e60396ce1e671d717a5d446460b571fef803.
Thanks!